megamek/src/megamek/common/Mech.java

```diff
@@ -171,20 +171,32 @@
     public static final int COCKPIT_QUADVEE = 13;
 
     public static final int COCKPIT_SUPERHEAVY_INDUSTRIAL = 14;
+    
+    public static final int COCKPIT_SUPERHEAVY_COMMAND_CONSOLE = 15;
+    
+    public static final int COCKPIT_SMALL_COMMAND_CONSOLE = 16;
+	
+    public static final int COCKPIT_INDUSTRIAL_COMMAND_CONSOLE = 17;	
```

Comment:
Delete industrial command console cockpit types.

Reply:
Thanks for the correction - I did it.